### PR TITLE
add platform specific tracking ID and suppress health check events

### PR DESCRIPTION
**Pull request by rmberrios**
add platform specific tracking ID

### CODE REVIEW

File: lib/voxa-ga.js [hunk 1]
```diff
@@ -12,13 +12,19 @@ const defaultConfig = {
...
+
+    const trackingId = _.get(pluginConfig, platformName) || pluginConfig.trackingId;
+    pluginConfig.trackingId = trackingId;
+
+    if (!pluginConfig.trackingId) throw new Error('trackingId is required in the config file');
+
     event.ga = new EventRider(event, pluginConfig);
-    event.ga.time("Interaction", "Request");
+    event.ga.time('Interaction', 'Request');
   });
```
Comment (rmberrios) [inline lib/voxa-ga.js:36]: Here is the change ^^ mainly on `onRequestStarted`

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "voxa-ga",
-  "version": "2.0.3",
+  "version": "2.1.0-alpha",
```
Comment (wuelcas) [inline package.json]: Any reason why not make a non alpha/beta release?

**Review by sloaisiga — COMMENTED** [review]
Any new test?

**Review by wuelcas — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (rmberrios) [reply package.json]: I like to test it out, and check there are no bugs with this change then I can make a proper release

**Comment by armonge** [comment]
I don't see any documentation or any new test

File: lib/EventRider.js [hunk 3]
```diff
@@ -24,9 +24,13 @@ module.exports = class GoogleAnalyticsEventRider {
       }
     );
 
+    let countryCode;
```
Comment (armonge) [inline lib/EventRider.js:27]: @rmberrios what's this change for?

File: package.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "voxa-ga",
-  "version": "2.0.3",
+  "version": "2.1.0-alpha3",
```
Comment (armonge) [inline package.json:3]: seems like this was a bug fix, not new functionality, as such should have been 2.0.4 and maybe some alpha instead of 2.1.0

Comment (rmberrios) [reply package.json:3]: add platform specific tracking ID - new functionality
fix geoId on Google Assistant - bug fix
remove health check event - bug fix

File: lib/EventRider.js (hunk 3 above)
Comment (rmberrios) [reply lib/EventRider.js:27]: https://github.com/VoxaAI/voxa-ga/pull/23/files/4ee24ea6b1f8898dc0a4b50205c4c9ecc3d49be4#diff-bf7e5bfd41e52774aa847a8a2521a8b4R33

If you don't define specific country languages on dialogflow, you won't receive a country code

**Pull request merged**